test/UnitTest/Microsoft.AspNet.OData.Test.Shared/UnicodeCharactersTest.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if NETCORE
+using System;
```

Comment:
Most of these look the same any reason for including them in different blocks

Reply:
@marabooy Fixed